### DISCUSSION TITLE
Specify API key security in environment variable?

### DISCUSSION THREAD

**Discussion by onionhammer** [qdrant/qdrant#2369] [General]
I see here it can be specified in a config file; is it possible to specify it in an environment variable?

https://qdrant.tech/documentation/guides/security/

It would be nice to be able to do that for some basic measure of security, while being able to use the default docker image and not have to mount a whole volume

**Comment by generall** [comment]
Yes https://qdrant.tech/documentation/guides/configuration/#environment-variables
